test/coverage-test/test/threshold-failure.test.ts

```diff
@@ -28,6 +28,35 @@ test('failing thresholds', async () => {
   expect(stderr).toContain('ERROR: Coverage for functions (25%) does not meet "**/fixtures/src/math.ts" threshold (100%)')
 })
 
+test('failing absolute thresholds', async () => {
+  const { exitCode, stderr } = await runVitest({
+    include: [normalizeURL(import.meta.url)],
+    coverage: {
+      all: false,
+      include: ['**/fixtures/src/math.ts'],
+      thresholds: {
+        '**/fixtures/src/math.ts': {
+          branches: -1,
+          functions: -2,
+          lines: -5,
+          statements: -1,
+        },
+      },
+    },
+  }, { throwOnError: false })
+
+  expect(exitCode).toBe(1)
+  if (isV8Provider()) {
+    expect(stderr).toContain(`ERROR: Uncovered lines (6) exceed "**/fixtures/src/math.ts" threshold (5)`)
+    expect(stderr).toContain(`ERROR: Uncovered functions (3) exceed "**/fixtures/src/math.ts" threshold (2)`)
+    expect(stderr).toContain('ERROR: Uncovered statements (6) exceed "**/fixtures/src/math.ts" threshold (1)')
+  }
+  else {
+    expect(stderr).toContain(`ERROR: Uncovered functions (3) exceed "**/fixtures/src/math.ts" threshold (2)`)
+    expect(stderr).toContain('ERROR: Uncovered statements (3) exceed "**/fixtures/src/math.ts" threshold (1)')
+  }
+})
```

Comment:
```suggestion
  expect(exitCode).toBe(1)

  if (isV8Provider()) {
    expect(stderr).toContain('ERROR: Uncovered lines (6) exceed "**/fixtures/src/math.ts" threshold (5)')
    expect(stderr).toContain('ERROR: Uncovered functions (3) exceed "**/fixtures/src/math.ts" threshold (2)')
    expect(stderr).toContain('ERROR: Uncovered statements (6) exceed "**/fixtures/src/math.ts" threshold (1)')
  }
  else {
    expect(stderr).toContain('ERROR: Uncovered functions (3) exceed "**/fixtures/src/math.ts" threshold (2)')
    expect(stderr).toContain('ERROR: Uncovered statements (3) exceed "**/fixtures/src/math.ts" threshold (1)')
  }
})
```

Reply:
One last small refactor and it's good to go.

Looks like I cannot apply this change myself, the `Maintainers are allowed to edit this pull request.` is not enabled on this PR. Probably due to origin of the PR being from `votingworks` organization. 🤔

Reply:
Yeah, I don't see any way to enable maintainers to edit in the UI or in these docs: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork